contracts/EscrowBase.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Base contract for escrows. Main idea: each mech type has its own respective escrow contract to verify payment rates and manage fees.

Reply:
The concept of the escrow means we have unnecessary transfers from Nevermined to here and then from here to withdrawal destination.